app/harvesters/Itms/accounting_documents_harvester.rb

```diff
@@ -0,0 +1,92 @@
+require 'harvester_utils/downloader'
+
+class Itms::AccountingDocumentsHarvester
+  include Itms::HarvesterHelpers
+
+  class ApiError < StandardError; end
+
+  attr_reader :min_id, :downloader
+
+  def self.run(min_id: 0, downloader: HarvesterUtils::Downloader)
+    new(min_id, downloader).run
+  end
+
+  def initialize(min_id, downloader)
+    @min_id = min_id
+    @downloader = downloader
+  end
+
+  def run
+    response = downloader.get("https://opendata.itms2014.sk/uctovneDoklady?min_id=#{min_id}&limit=100")
+    raise(ApiError, 'API not available') if response.response_code != 200
+    json_response = JSON.parse(response.body)
+
+    json_response.each do |json|
+      Itms::AccountingDocument.transaction do
+        begin
+          document = Itms::AccountingDocument.new
```

Comment:
fair point. sam som nazoru, ze by nemala existovat metoda, ktora ma viac ako ~5 riadkov. ale, "make it works, make it nice, make it fast" a aktualne sme stale v tej prvej faze. aktualne este nieje dotiahnuta samotna logika stahovania dokonca preto si myslim, ze hranie sa s zapuzdrovanim je zbytocne.